### DISCUSSION TITLE
Elasticsearch sigma ealastalert to thehive4

### DISCUSSION THREAD

**Discussion by adilraad2001** [jertel/elastalert2#1455] [Q&A] [answered]
Hello guys
can someone tell me how i can use   sigma rules 
to anaylse elasticsearch and send it to thehive by elastalert2

**Comment by jertel** [comment, marked as answer]
The [docs](https://elastalert2.readthedocs.io/en/latest/alerts.html#thehive) explain how to setup ElastAlert 2 with TheHive. The other areas in your questions are out of scope of this project. You might want to check out [Security Onion](https://securityonion.com), as that project already handles most of what you're asking.
